### PR TITLE
[tests-only][full-ci]Bump ocis commit id for tests

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR bumps ocis commit id

## Related Issue

- part of: https://github.com/owncloud/QA/issues/767

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles5</strong> https://drone.owncloud.com/owncloud/web/28782/60/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesActionMenu-fileFolderActionMenu_feature-L14.png
  


</p></details>

**Comment by SwikritiT** [comment]
> Results for **oCISFiles5** https://drone.owncloud.com/owncloud/web/28782/60/1
> 
>  The acceptance tests failed on retry. Please find the screenshots inside ...

passing locally. I'll restart and see if it passes or not

**Comment by SwikritiT** [comment]
> Results for **oCISFiles5** https://drone.owncloud.com/owncloud/web/28789/60/1
> 
>  The acceptance tests failed on retry. Please find the screenshots inside ...

This shows that the `preview` is displayed twice. But I cannot reproduce it locally.

**Comment by SwikritiT** [comment]
> > Results for **oCISFiles5** https://drone.owncloud.com/owncloud/web/28789/60/1
> > The acceptance tests failed on retry. Please find the screenshots inside ...
> 
> This shows that the `preview` is displayed twice. But I cannot reproduce it locally.

Finally could reproduce locally the issue is here https://github.com/owncloud/ocis/issues/4771

**Comment by SwikritiT** [comment]
Do not merge until the reply from https://github.com/owncloud/web/pull/7757#issuecomment-1273164234

File: dev/docker/ocis.web.config.json [hunk 1]
```diff
@@ -27,7 +27,6 @@
   "apps": [
     "files",
     "text-editor",
-    "preview",
```
Comment (SwikritiT) [inline dev/docker/ocis.web.config.json:30]: for more context on why this is removed https://github.com/owncloud/ocis/issues/4771#issuecomment-1273222856

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**